products/green_fast_track/models/_sources.yml

```diff
@@ -126,12 +126,11 @@ sources:
         columns:
           - name: bbl
           - name: last_actio
-          - name: latitude
           - name: lm_name
           - name: lm_type
-          - name: longitude
           - name: most_curre
           - name: status
+          - name: wkb_geometry
 
```

Comment:
`latitude` & `longitude` columns no longer exist at the source - remove. These columns are actually not used in the build, so no impact. Added `wkb_geometry` bc it's actually expected.